lib/blot_axis.c

```diff
@@ -1,5 +1,5 @@
 /* blot: internal object that calculates axis tick/text placement */
```

Comment:
@coderabbitai generate unit tests for this file.

Reply:
add ctest to this project.  add google test framework.  add tests in tests directory